framework/src/org/checkerframework/framework/type/TypeFromMemberVisitor.java

```diff
@@ -27,8 +27,7 @@
     @Override
     public AnnotatedTypeMirror visitVariable(VariableTree node,
                                              AnnotatedTypeFactory f) {
-        AnnotatedTypeMirror result = f.fromTypeTree(node.getType());
-        result.clearAnnotations();
+        AnnotatedTypeMirror result = f.type(node.getType());
```

Comment:
I think this is one of the few uses of `fromTypeTree`. I think there is some complication why you need to know whether the tree is a type tree or not.